tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -63,6 +101,14 @@ Create VCH - custom base disk
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Create VCH - Folder Structure Correctness
+    Install VIC Appliance To Test Server
+    ${rc}  ${out}=  Run And Return Rc And Output  govc ls vm | grep %{VCH-NAME}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${out}  vm/%{VCH-NAME}
+    Check VM Folder Path  %{VCH-NAME}
+    Cleanup VIC Appliance On Test Server
```

Comment:
I would suggest calling all cleanup functions using `TEARDOWN` (either test or suite level based on need), so to make sure they are run in any conditions instead of manually we doing it which is error prone.

Reply:
I agree, in this case there are many more functions that perform cleanup in this way besides this one. I think it would be more valuable to leave this pattern(as it is the same way the other tests do it). That way a more broad effort on master can be made to address the way we clean up VCH's in the `Group6` category.